db/migrate/2019031920190319163755_add_provider_ucas_preferences.rb

```diff
@@ -0,0 +1,11 @@
+class AddProviderUCASPreferences < ActiveRecord::Migration[5.2]
+  def change
+    create_table :provider_ucas_preference do |t|
+      t.integer :provider_id, null: false
+      t.text :name, null: false
+      t.text :value
```

Comment:
I'm not sure I'm a big fan of `value` as a name for an attribute, very ambiguous.

Reply:
Ya, I decided this whole abstraction was unnecessary, and this approach has been redone.